src/zc/buildout/buildout.py

```diff
@@ -1777,27 +1777,32 @@ def _check_for_unused_options_in_section(buildout, section):
 
 Options:
 
-  -h, --help
+  -c config_file
 
-     Print this message and exit.
+     Specify the path to the buildout configuration file to be used.
+     This defaults to the file named "buildout.cfg" in the current
+     working directory.
 
-  --version
+  -D
 
-     Print buildout version number and exit.
+    Debug errors.  If an error occurs, then the post-mortem debugger
+    will be started. This is especially useful for debuging recipe
+    problems.
```

Comment:
Some of these text sections are indented with 4 spaces, and some are indented with 5 spaces.  This bugs me. ;)